src/main/java/Duke.java

```diff
@@ -1,10 +1,148 @@
+            case "todo":
+                taskDescription = userDescription;
+                t = new Task(taskDescription);
+                break;
+            case "deadline":
+                parsedUserDescription = new SplitString(userDescription, "/by ");
+                taskDescription = parsedUserDescription.getFirstString();
+                String taskBy = parsedUserDescription.getSecondString();
+                t = new Deadline(taskDescription, taskBy);
+                break;
+            case "event":
+                parsedUserDescription = new SplitString(userDescription, "/at ");
+                taskDescription = parsedUserDescription.getFirstString();
+                String taskAt = parsedUserDescription.getSecondString();
+                t = new Event(taskDescription, taskAt);
+                break;
+            default:
+                throw new IllegalStateException("Unexpected value: " + userCommand);
+        }
+
+        tasks[numTasks] = t;
+        String addTaskMessage = "Got it. I've added this task: \n   " + tasks[numTasks].toString() + "\n";
+        numTasks++;
+        addTaskMessage += "Now you have " + String.valueOf(numTasks) + " task";
+        addTaskMessage += numTasks == 1 ? "" : "s";
+        addTaskMessage += " in the list.";
+        printMessage(addTaskMessage);
+    }
+
+    public static void doTask(String userDescription) {
+        int taskNum = Integer.parseInt(userDescription);
+        Task t = tasks[taskNum - 1];
+        t.markAsDone();
+        String doTaskMessage = "Nice! I've marked this task as done: \n" + "   " + t.toString();
+        printMessage(doTaskMessage);
+    }
+
+    public static void deleteTask(String userDescription) {
+        int taskNum = Integer.parseInt(userDescription);
+        Task t = tasks[taskNum - 1];
+        String deleteTaskMessage = "Noted, I've removed this task: \n" + "   " + t.toString() + "\n";
+
+        int i = taskNum;
+        while (i < 99) {
+            tasks[i - 1] = tasks[i];
+            i++;
+        }
+
+        numTasks--;
+        deleteTaskMessage += "Now you have " + String.valueOf(numTasks) + " task";
+        deleteTaskMessage += numTasks == 1 ? "" : "s";
+        deleteTaskMessage += " in the list.";
+        printMessage(deleteTaskMessage);
+    }
+
+    public static void main(String[] args) {
+        printWelcomeMessage();
+
+        String userInput, userCommand, userDescription;
+        SplitString parsedUserInput;
+        Scanner scan = new Scanner(System.in);
+
+        while (true) {
+            userInput = scan.nextLine();
+            parsedUserInput = new SplitString(userInput, " ");
+            userCommand = parsedUserInput.getFirstString().toLowerCase();
+            userDescription = parsedUserInput.getSecondString();
+
+            if (String.valueOf(userCommand).equals("bye") == true) {
+                break;
+            }
+            switch (userCommand) {
+                case "list":
```

Comment:
The 'case' statements inside a 'switch' statement should be aligned to the same indentation as the 'switch' statement.

If IntelliJ is your IDE, you can follow the instructions at the following link for how to get IntelliJ to follow this coding convention for you.
https://stackoverflow.com/questions/40888798/config-intellij-switch-case-style#:~:text=Go%20to%20Settings%20%3E%20Editor%20%3E%20Code,Indent%20'case'%20branches%22.


For example:
```
switch (userCommand) {
case "list":
  // Code
  break;
case "done":
  // Code
  break;
...

```